src/components/PasswordForm.js

```diff
@@ -29,7 +29,10 @@ function PasswordForm(props) {
         const passwordObject = {
             password: enteredPassword
         };
-        console.log(passwordObject.password)
+        console.log('>>>entered Password:', passwordObject.password);
+    
+        sessionStorage.setItem('storedPassword', passwordObject.password)
```

Comment:
This is ok, but it would be making more sense if we save only when the API call is successful. Because this input password can be the wrong one and we don't need to save it in that case.